android_parser.py

```diff
@@ -13,4 +16,22 @@ def parse_args(self):
             required=True
         )
 
+
+        teststring = ' '.join(
+            ['({}) {}'.format(str(i), name) for i, name in enumerate(TESTS)]
+        )
+        args = {
+            'help': 'Test to run. '
+                'These options are curently available: %s' % teststring,
```

Comment:
Not a nit, just a suggestion: 'Currently available options are: %s'

Reply:
Removed in latest commit.